src/core/annotation.js

```diff
@@ -494,6 +496,11 @@ class AnnotationBorderStyle {
     }
   }
 
+  /**
```

Comment:
Since this is only for documentation/typing purposes, could we perhaps find a way to move these kind of generic typings to a central place, for example https://github.com/mozilla/pdf.js/blob/master/src/doc_helper.js? The same applies to other of these blocks below.